src/metadata-stream.js

```diff
@@ -6,12 +6,24 @@
 (function(window, videojs, undefined) {
   'use strict';
   var
```

Comment:
@mmcc I'm with you-- I definitely want to adopt [standard](https://github.com/videojs/standard) across all the contrib plugins. We haven't had a chance to convert this project over, however, so I'm following the current conventions until we can do that sweep.